apps/web/composables/useHomepage/useHomepage.ts

```diff
@@ -38,6 +42,14 @@ export const useHomepage: UseHomepageDataReturn = () => {
     state.value.loading = false;
   };
 
+  watch(
+    () => currentLocale.value,
+    async (differentLang) => {
+      currentLocale.value = differentLang;
```

Comment:
```suggestion
    async (newLocale) => {
      currentLocale.value = newLocale;
```
We should use consistent wording, so locale over language here.

I also prefer "new" here to indicate that it's the locale the user switched to.

Reply:
I also wanted to go with new but I get a Lint error when using new

Reply:
Looking at the [rule description](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-keyword-prefix.md), I don't think that rationale applies in this case. I think we should `ignore` the rule and go with `newLocale`.